### ISSUE TITLE
Having some issues getting up and running

### ISSUE THREAD

**Issue by danjuggles** [scmbreeze/scm_breeze#240]
## Problem
When I run any of the scm_breeze commands I am getting strange functionality..

Any help would be greatly appreciated, as I am very keen to get this working 😄 

### Command: `gs`
Note: `git status` works normally

```
The system cannot find the path specified.
C:/Users/dan.evans/.scm_breeze/lib/git/status_shortcuts.rb:29:in `<main>': undefined method `[]' for nil:NilClass (NoMethodError)
On branch master
nothing to commit, working tree clean

There were more than 150 changed files. SCM Breeze has fallen back to standard `git status` for performance reasons. 
```

### Command: `gb` or `git branch`

```
 $ git branch
-e:1:in ``': No such file or directory - /mingw64/bin/git branch --color=always  (Errno::ENOENT)
        from -e:1:in `<main>'
```

### OS:
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.14393 N/A Build 14393

### Shell:
```
$ bash --version
GNU bash, version 4.4.12(1)-release (x86_64-pc-msys)
```

### scm_breeze

```
$ git show -n 1
commit a59ace0f297402031d3a47b1b59f9bf8c776a822 (HEAD -> master, origin/master, origin/HEAD)
Author: Nathan Broadbent <ndbroadbent@users.noreply.github.com>
Date:   Tue Sep 26 02:47:48 2017 +0700

    Changed logo
```

**Comment by danjuggles** [comment]
I have this line in my `.bash_profilfe`

 `[ -s "/c/Users/dan.evans/.scm_breeze/scm_breeze.sh" ] && source "/c/Users/dan.evans/.scm_breeze/scm_breeze.sh" `

**Comment by ghthor** [comment]
What version of ruby do you have installed?

**Comment by danjuggles** [comment]
```
ruby -v
ruby 2.4.2p198 (2017-09-14 revision 59899) [x64-mingw32]
```

**Comment by ghthor** [comment]
Alright, I'm checking if that version of ruby breaks my local installation, caveat I'm on OSX.
Which version of `git` do you have installed?

**Comment by ghthor** [comment]
Could you try this changeset to see if this fixes the issue with `gb`? https://github.com/scmbreeze/scm_breeze/pull/232/files

**Comment by danjuggles** [comment]
Thank you very much @ghthor! I have made the changes as per your link, and now scm_breeze seems to be working, I very much appreciate the help

**Comment by ghthor** [comment]
Fixed by #232 which still needs to be incorporated properly into mainline. Thanks @danjuggles for juggling this around!
